### PR TITLE
Add the 1.2.10 Release Note

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the 1.2.10 release note.

> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: downloads/release-notes/1.2.10/RELEASE_NOTE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- jBallerina 1.2.0 but switched to a previous version: `ballerina dist pull 1.2.10`
+- a jBallerina version below 1.1.0: install via the [installers](/downloads/)
+
+**For new users:**
+
+If you have not installed jBallerina, then download the [installers](/downloads/) to install.
+
+
+
+
```
Comment (praveennadarajah) [inline downloads/release-notes/1.2.10/RELEASE_NOTE.md:28]: Do we need these extra lines? If not shall we remove it?

Comment (praneesha) [reply downloads/release-notes/1.2.10/RELEASE_NOTE.md]: Done.

**Review by praveennadarajah — APPROVED** [review]

File: downloads/release-notes/1.2.10/RELEASE_NOTE.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+layout: ballerina-blank-page
+title: Release Note
+---
+### Overview of jBallerina 1.2.10
+
+The jBallerina 1.2.10 patch release improves upon the 1.2.9 release by introducing the features listed below and addressing a number of [bugs](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A%22Ballerina+1.2.10%22+label%3AType%2FBug+is%3Aclosed) and [improvements](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A%22Ballerina+1.2.10%22+is%3Aclosed+label%3AType%2FImprovement).
```
Comment (anupama-pathirage) [inline downloads/release-notes/1.2.10/RELEASE_NOTE.md:7]: It seems we need to remove the phrase `introducing the features listed below` and `improvements` as we don't have any.

Comment (praneesha) [reply downloads/release-notes/1.2.10/RELEASE_NOTE.md:7]: Done.

**Review by anupama-pathirage — APPROVED** [review]

**Pull request merged**